app/controllers/contestant_project_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ContestantProjectController < ApplicationController
+  def create
+    # ContestantProject.create(contestant_id: params[:id], project_id: @lit_fit.id)
```

Comment:
This was headed in the right direction. You may have been looking for:

`ContestantProject.create(contestant_id: params[:contestant_id], project_id: params[:project_id)`

You URL that you may have been looking for was `POST projects/:project_id/contestants'